source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -136,7 +136,92 @@ The web console displays a widget for tracking the installation progress.
 
 Once installation completes, click :guilabel:`View Operator` to view the MinIO Operator page. 
 
-3) Open the MinIO Operator Interface
+3) Configure TLS Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have installed the MinIO Operator from Red Hat OperatorHub, the installation process also configures the :openshift-docs:`OpenShift Service CA Operator <security/certificate_types_descriptions/service-ca-certificates.html>`.
+This Operator manages the the TLS certificate required to access the MinIO Operator Console and Tenants.
```

Comment:
I am fuzzy whether this is being able to see/act on Tenants, or all of Operator entirely.